docs/GettingStarted.asciidoc

```diff
@@ -153,8 +153,15 @@ following list.
   that workaround needles are in place, a softfailure has been recorded explicitly
   via +record_soft_failure+ (from os-autoinst) or a job failure has been ignored
   explicitly via a <<UsersGuide.asciidoc#https://github.com/os-autoinst/openQA/blob/master/docs/UsersGuide.asciidoc#show-bug-or-label-icon-on-overview-if-labeled-gh550,job label>>.
-* *incomplete* The job is no longer running but no result was provided. Either
-  it was cancelled while running or it crashed.
+* *timeout_exceeded* The job has been aborted because +MAX_JOB_TIME+ which is by default two
```

Comment:
I'd suggest reshuffling this to `The job was aborted because +MAX_JOB_TIME+ was exceeded, which is two hours by default.`

You'll want to use the simple past tense in all of following sentences as well, since you're explaining the final state of the job and not something that just happened.